.github/workflows/test.yml

```diff
@@ -18,10 +18,10 @@ jobs:
       run: sudo apt-get install libleptonica-dev libtesseract-dev clang tesseract-ocr-eng
     - uses: actions/checkout@v2
     - name: Build
-      run: cargo build --verbose
+      run: cargo build --verbose --no-default-features
```

Comment:
Fix build failures that depend on a newer version of tesseract

https://github.com/houqp/leptess/actions/runs/7589902627/job/20675452376